src/sentry/static/sentry/app/components/alertMessage.jsx

```diff
@@ -36,7 +36,7 @@ export default class AlertMessage extends React.PureComponent {
   static propTypes = {
     alert: PropTypes.shape({
       id: PropTypes.string,
-      message: PropTypes.string.isRequired,
+      message: PropTypes.node.isRequired,
```

Comment:
Should this be node or string?

Reply:
node, so you can add things like links.

Reply:
But if it was a string before, could this change cause prop type warnings in other uses?

Reply:
`node` encompasses string